pulp_2to3_migration/app/pre_migration.py

```diff
@@ -178,7 +178,8 @@ def pre_migrate_content_type(content_model, mutable_type, lazy_type, premigrate_
                     pulp2_last_updated=record._last_updated,
                     pulp2_storage_path=record._storage_path,
                     downloaded=downloaded,
-                    pulp2_repo=relation.pulp2_repository
+                    pulp2_repo=relation.pulp2_repository,
+                    pulp2_subid=relation.pulp2_repository.pk,  # May not need this with correct UQ
```

Comment:
See discussion under migration 0024. 

Assuming we decide we still need to fill subid with repo_id for errata: this code is not plugin-aware and can't have an "if this is an erratum" switch.  The only reason what I'm doing here is OK is because the only thing that sets set_pulp2_repo to true is Pulp2Erratum. I don't like implicit assumptions. We can add a set_subid flag, we can rename set_repo_id to set_repo_and_sub_id, we can rename it to set_sub_id and drop the repo_id column.